test/spec/tokenManager.js

```diff
@@ -307,13 +287,13 @@ define(function(require) {
       it('automatically refreshes a token by default', function(done) {
         var expiresAt = tokens.standardIdTokenParsed.expiresAt;
         return oauthUtil.setupFrame({
-          fastForwardToTime: expiresAt + 1,
+          authClient: setupSync({
+            autoRefresh: true
```

Comment:
The spec name is "automatically refreshes tokens by default", but we're setting it to `true` here.